### ISSUE TITLE
Debug error! Program: ... abort() has been called

### ISSUE THREAD

**Issue by insinfo** [jnschulze/flutter-webview-windows#86]
I'm getting this error all the time intermittently. Every time I open a screen of the app that has the webview, sometimes it opens normally and sometimes this error appears
it looks like it's just running a hot reload that the error appears
since it presents this error, if I click on any of the buttons the application unexpectedly closes

```dart
import 'dart:io';

import 'package:flutter/material.dart';
import 'package:webview_flutter/webview_flutter.dart' as mobWebview;
import 'package:webview_windows/webview_windows.dart' as winWebview;

class IptuHomePage extends StatefulWidget {
  @override
  _IptuHomePageState createState() => _IptuHomePageState();
}

class _IptuHomePageState extends State<IptuHomePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Container(
        child: AspectRatio(
          aspectRatio: 1 / 1.4142,
          child: CustomWebView(
            html: '<body><h1>test</h1></body>',
          ),
        ),
      ),
    );
  }
}

class CustomWebView extends StatefulWidget {
  final String html;
  final String userAgent;

  const CustomWebView({
    Key? key,
    required this.html,
    this.userAgent =
        'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36',
  }) : super(key: key);
  @override
  _CustomWebViewState createState() => _CustomWebViewState();
}

class _CustomWebViewState extends State<CustomWebView> {
  winWebview.WebviewController winWebviewController =
      winWebview.WebviewController();

  late mobWebview.WebViewController mobWebviewController;

  @override
  void initState() {
    super.initState();
    init();
  }

  bool isLoading = true;
  Future<void> init() async {
    if (Platform.isWindows) {
      await winWebviewController.initialize();
      await winWebviewController.setUserAgent(widget.userAgent);
      await winWebviewController.loadStringContent(widget.html);
      setState(() {
        isLoading = false;
      });
    }
  }

  @override
  Widget build(BuildContext context) {
    if (Platform.isWindows) {
      return isLoading
          ? Center(
              child: CircularProgressIndicator(),
            )
          : windows();
    } else {
      return mobile();
    }
  }

  Widget mobile() {
    return mobWebview.WebView(
      zoomEnabled: true,
      javascriptMode: mobWebview.JavascriptMode.unrestricted,
      initialUrl: 'about:blank',
      onWebViewCreated: (mobWebview.WebViewController webViewController) {
        mobWebviewController = webViewController;
        mobWebviewController.loadHtmlString(widget.html);
      },
    );
  }

  Widget windows() {
    return winWebview.Webview(
      winWebviewController,
    );
  }

  @override
  void dispose() {
    winWebviewController.dispose();
    super.dispose();
  }
}



```

![image](https://user-images.githubusercontent.com/12227024/150356270-385ee18c-ba96-4ce0-bf20-18df1291ad2f.png)

![image](https://user-images.githubusercontent.com/12227024/150354399-a7e5989d-19c0-4ebe-bcad-5874fc780ea4.png)

![image](https://user-images.githubusercontent.com/12227024/150359865-a5617618-f12a-44cf-89bf-fd651b4cb835.png)

![image](https://user-images.githubusercontent.com/12227024/150361035-5d495949-e270-4707-9f2c-add450118388.png)

**Comment by jnschulze** [comment]
What’s your webview_windows version? Is the most recent version (0.1.4) failing as well? If so, does 0.1.0 work for you?

**Comment by RookieApe** [comment]
> What’s your webview_windows version? Is the most recent version (0.1.4) failing as well? If so, does 0.1.0 work for you?

Me too，0.1.0 is ok， 0.1.4 not

**Comment by insinfo** [comment]
@RookieApe @jnschulze 
yes it seems to be working normally in version 0.1.0

**Comment by jnschulze** [comment]
Should be fixed in v0.1.5.
